### PR TITLE
Cc/styling tweaks

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
#3672 

#### What's this PR do?
Styling updates per Ferdi's feedback. See commit messages or https://mit.enterprise.slack.com/files/W010W1KTE73/F03SWRZ8S6L/https-discussions-rc-odl-mit-edu-infinite-fields-biology-.png

#### How should this be manually tested?
- Check that the pages work and look like they do below. Keep an eye out for any styling oddities.
- Check that the field page title links to the field page itself, similar to open-discussions channels

#### Screenshots (if appropriate)

**Notes:**
- Search page still needs styling, my edits there were fairly small.
- Regarding avatar colors, see https://github.com/mitodl/hq/discussions/194

<table>
    <thead></thead>
        <tr>
            <th></th>
            <th>Mobile</th>
            <th>Desktop</th>
        </tr>
    <tbody>
        <tr>
            <th>Home</th>
            <td>
                home_mobile
            </td>
            <td>
                home_wide
            </td>
        </tr>
        <tr>
            <th>View Field</th>
            <td>
                field_mobile
            </td>
            <td>
                <p>⚠️ Note: there's a right-hand column with no content at the moment (for widgets), which is why it looks un-centered.</p>
                field_wide
            </td>
        </tr>
        <tr>
            <th>Manage Field</th>
            <td>
                manage_mobile
            </td>
            <td>
                manage_wide
            </td>
        </tr>
        <tr>
            <th>Search</th>
            <td>
                search_mobile
            </td>
            <td>
                search_wide
            </td>
        </tr>
    </tbody>
</table>

### CODE REVIEW

**Comment by ChristopherChudzicki** [comment]
@pdpinch @Ferdi Screenshots above for your perusal

**Comment by Ferdi** [comment]
looks great.

183092047-6a672d26-7f67-4148-a31e-27e9b642975f-png-2656×1588-


The styling of the card in search looks a bit diffrent from the cards in the field page ?

**Comment by ChristopherChudzicki** [comment]
The search and field page styling is the same for the two cards as far as I can tell.

The search cards seem to be missing the date chip. @abeglova or I can look into that separate from this PR.

File: frontends/infinite-corridor/src/pages/FieldPage.tsx [hunk 1]
```diff
@@ -94,48 +93,66 @@ const FieldCarousel: React.FC<FieldListProps> = ({ list }) => {
...
                 {fieldLists.map(list => (
                   <FieldList key={list.id} list={list} />
                 ))}
               </TabPanel>
-              <TabPanel value="about">BBBBBB</TabPanel>
-            </Grid>
-            <Grid item xs={12} sm={3}>
-              Featured Video
+              <TabPanel value="about" className="page-nav-content">
+                At vero eos et accusamus et iusto odio dignissimos ducimus qui
```
Comment (mbertrand) [inline frontends/infinite-corridor/src/pages/FieldPage.tsx:141]: Remove this?

**Review by mbertrand — APPROVED** [review]
LGTM, there's some sample text for "About" tab in the PR, not sure if that was left there deliberately as a placeholder or not.  👍

File: frontends/infinite-corridor/src/pages/FieldPage.tsx (hunk 1 above)
Comment (ChristopherChudzicki) [reply frontends/infinite-corridor/src/pages/FieldPage.tsx:141]: I did put it there deliberately, but probably better to just leave it blank.

**Pull request merged**